### PR TITLE
ssh: add -t in example description

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/common/ssh.md [hunk 1]
```diff
@@ -15,15 +15,15 @@
 
 `ssh {{username}}@{{remote_host}} -p {{2222}}`
 
-- Run a command on a remote server:
+- Run a command on a remote server (use `-t` to force a [t]ty allocation allowing interaction with the remote command):
```
Comment (sbrl) [inline pages/common/ssh.md:18]: Hmmm. I'm on the fence about this one, as it seems like it's essentially a second example.

Comment (marchersimon) [reply pages/common/ssh.md:18]: I also think this should be either a separate example or be removed

File: pages/common/ssh.md [hunk 2]
```diff
@@ -15,15 +15,15 @@
...
 
-`ssh {{remote_host}} {{command -with -flags}}`
+`ssh {{username}}@{{remote_host}} {{command}} {{command_arguments}}`
 
 - SSH tunneling: Dynamic port forwarding (SOCKS proxy on localhost:1080):
 
 `ssh -D {{1080}} {{username}}@{{remote_host}}`
 
-- SSH tunneling: Forward a specific port (localhost:9999 to example.org:80) along with disabling pseudo-[t]ty allocation and executio[n] of remote commands:
+- SSH tunneling: Forward a specific port (localhost:9999 to example.org:80) along with disabling pseudo-[T]ty allocation and executio[N] of remote commands:
```
Comment (marchersimon) [inline pages/common/ssh.md:26]:
```suggestion
- SSH tunneling: Forward a specific port (`localhost:9999` to `example.org:80`) along with disabling pseudo-[T]ty allocation and executio[N] of remote commands:
```

**Review by marchersimon — APPROVED** [review]
Looks good

**Pull request merged**